pkgs/development/python-modules/jug/default.nix

```
@@ -21,6 +22,9 @@ buildPythonPackage rec {
     sha256 = "859a4b4cb26a0010299b189c92cfba626852c97a38e22f3d1b56e4e1d8ad8620";
   };
 
+  checkInputs = [ pytest pytestCheckHook ];
```

checkhook will bring in pytest, shouldn't need both
```suggestion
  checkInputs = [ pytestCheckHook ];
```